### PR TITLE
Pylint errors 1

### PR DESCRIPTION
**Pull request by DanielNoord**
- [x] Add yourself to ``CONTRIBUTORS.txt`` if you are a new contributor.
- [x] Write a good description on what the PR does.

## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

## Description

Code examples based on [pylint-errors](https://github.com/vald-phoenix/pylint-errors) and my own work.

Ref. #5953.

### CODE REVIEW

File: doc/data/messages/a/abstract-method/bad.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class AnotherClass:
+    @abc.abstractmethod
+    def my_abstract_method(self):
+        pass
+
+
+class AnotherParentClass(AnotherClass):  # [abstract-method]
+    pass
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/a/abstract-method/bad.py:20]:
```suggestion
class Pet:
    def make_sound(self):
        raise NotImplementedError


class Cat(Pet):  # [abstract-method]
    pass


import abc


class WildAnimal:
    @abc.abstractmethod
    def make_sound(self):
        pass


class Panther(WildAnimal):  # [abstract-method]
    pass
```

File: doc/data/messages/a/abstract-method/good.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+class AnotherClass:
+    @abc.abstractmethod
+    def my_abstract_method(self):
+        pass
+
+
+class AnotherParentClass(AnotherClass):
+    def my_abstract_method(self):
+        pass
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/a/abstract-method/good.py:22]:
```suggestion
class Pet:
    def make_sound(self):
        raise NotImplementedError


class Cat(Pet):
    def make_sound(self):
        print("Meeeow")


import abc


class WildAnimal:
    @abc.abstractmethod
    def make_sound(self):
        pass


class Panther(WildAnimal):
    def make_sound(self):
        print("MEEEOW")
```

**Review by Pierre-Sassoulas — CHANGES_REQUESTED** [review]
Some propositions. Btw in ``class AnotherParentClass(AnotherClass):`` I think the child class is actually ``AnotherParentClass`` and the mother class is ``AnotherClass``.

**Review by Pierre-Sassoulas — APPROVED** [review]

**Comment by DanielNoord** [comment]
Something weird going on with the CI for this PR. Second time it doesn't fire..

**Pull request merged**